### ISSUE TITLE
Bug with `io.Fonts.Fonts` in 1.89.3

### ISSUE THREAD

**Issue by pkdawson** [mellinoe/ImGui.NET#385]
The `io.Fonts.Fonts` vector remains empty after adding a font. The attached program outputs `0` with 1.89.3, and `1` with 1.89.2 or earlier.

It's simply:
```csharp
using System;
using ImGuiNET;

ImGui.CreateContext();
var io = ImGui.GetIO();
io.Fonts.AddFontDefault();
Console.WriteLine(io.Fonts.Fonts.Size);
```

I can't reproduce this when building ImGui.NET from source, just with the NuGet package, so I've had trouble finding the cause.

[ign-font-bug.zip](https://github.com/mellinoe/ImGui.NET/files/10564718/ign-font-bug.zip)

**Comment by zaafar** [comment]
what's wrong with having Fonts.Fonts.Size 0?
`io.Fonts.AddFontDefault();` isn't required by imgui anymore as shown here https://github.com/ocornut/imgui/blob/master/examples/example_win32_directx12/main.cpp#L100.

**Comment by pkdawson** [comment]
It's just a minimal example for reproduction purposes. The problem also exists when adding TTF fonts.

I need to be able to iterate over the loaded fonts, like you can in C++. It worked fine in ImGui.NET until 1.89.3.

**Comment by mellinoe** [comment]
Is this behavior different from the same code called in C++?

**Comment by zaafar** [comment]
> I can't reproduce this when building ImGui.NET from source, just with the NuGet package, so I've had trouble finding the cause.

I have reproduced it when using source as well.

> Is this behavior different from the same code called in C++?

let me test that.

**Comment by zaafar** [comment]
yeah looks like its only happening in c# (as in c++ tag v1.89.2) the Font Size increased to 1.
![image](https://user-images.githubusercontent.com/5079646/216783657-04c94bce-6d4d-461f-a4be-17ea8a037cd6.png)

**Comment by zaafar** [comment]
PR merged, going to release 1.89.4.
